src/js/hooks/cycleCount/useProductsTabExport.js

```diff
@@ -43,7 +43,7 @@ const useProductsTabExport = ({
         startDate: dateWithoutTimeZone({
           date: filters.startDate,
         }),
-        products: filters.products?.map((product) => product.id),
+        products: filters.products ? filters.products?.map((product) => product.id) : undefined,
```

Comment:
you could also just add a nullsafe after map
```suggestion
        products: filters.products?.map?.((product) => product.id)
```